### DISCUSSION TITLE
Prevent nodes collapsing when jumping between files

### DISCUSSION THREAD

**Discussion by sid-6581** [nvim-neo-tree/neo-tree.nvim#700] [Q&A]
If I jump between buffers that are in separate subdirectories and I have `follow_current_file` enabled, all the directories except the one the currently focused file is in will close. Is there a way to prevent that and just expand whatever is needed for the focused file while leaving the rest of the tree intact?

**Comment by cseickel** [comment]
There is no option for this, but if someone wanted to add it they would just need to create a config option for the choice and then use it to skip this line if desired:
https://github.com/nvim-neo-tree/neo-tree.nvim/blob/8238865e1d9c61f1a260c290653f2c419503e0a9/lua/neo-tree/sources/filesystem/init.lua#L96

**Reply by sid-6581** [reply]
Thanks for the quick reply! If you would accept a PR for that I'd be happy to contribute. The file following is very useful, but I also like to not lose track of where I've been.

**Reply by cseickel** [reply]
I would definitely accept that PR.
